src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php

```diff
@@ -67,38 +75,64 @@ public function buildSettings(SettingsBuilderInterface $builder)
     public function buildForm(FormBuilderInterface $builder)
     {
         $builder
-            ->add('title', 'text', array(
-                'label'       => 'sylius.form.settings.general.title',
-                'constraints' => array(
-                    new NotBlank()
+            ->add(
```

Comment:
I don't think it is so important, as both ways is standards. but i will rework in way that is already in file